tests/tensorflow/test_keras_model_export.py

```diff
@@ -600,7 +600,10 @@ def test_load_without_save_format(tf_keras_model, model_path):
 
 
 @pytest.mark.skipif(
-    not (_is_importable("transformers") and Version(tf.__version__) >= Version("2.6.0")),
+    # TODO: Reenable this test on TF 2.11.0 once a compatible version of the transformers
+    # library is released
+    Version(tf.__version__) == Version("2.11.0")
```

Comment:
This applies to cross version tests *and* master tests

Reply:
nice!